src/com/tictacgo/data/Piece.java

```diff
@@ -206,7 +205,7 @@ public Player getPlayer() {
 
     public void setPlayer(Player player) {
         this.player = player;
-        updateImageResource();
+        updateImageResource(true);
```

Comment:
I think this has some unmerged changes from other PRs? This was updated to be two separate updateImageResource methods.